src/firecrest/filesystem/transfer/models.py

```diff
@@ -68,3 +68,23 @@ class MoveRequest(FilesystemRequestBase):
 
 class MoveResponse(CamelModel):
     transfer_job: TransferJob
+
+
+class CompressRequest(FilesystemRequestBase):
+    target_path: str
+    account: Optional[str] = None
+    pattern: Optional[str] = None
```

Comment:
Question: should we name the new param **matchPattern**? Or something that makes it more indicative of the effect of the param.

Reply:
Yes, I think we can use `matchPattern`.

Reply:
actually, the best is to use `match_pattern`